packages/graphql-hooks/src/GraphQLClient.js

```diff
@@ -90,14 +90,20 @@ class GraphQLClient {
   }
   /* eslint-enable no-console */
 
-  generateResult({ fetchError, httpError, graphQLErrors, data }) {
+  generateResult({
+    fetchError,
+    httpError,
+    graphQLErrors,
+    data,
+    responseReducer = null
```

Comment:
Is the default value necessary here?

Reply:
no, not needed. just a habit of mine i'd say. can remove it if needed 👍

Reply:
Not a big deal, i only think that removing it would make the style more consinstent with the rest